python/Makefile

```diff
@@ -0,0 +1,13 @@
+ifeq ($(CI), true)
+    PYTEST_ARGS = -x -v --durations=30
```

Comment:
Are we sure we want early exit? I kind of like to know all the failures, especially if it's a weird platform.

Reply:
Durations is cool though 👍

Reply:
Just copied these over. I'm down to remove the -x

Reply:
updated